src/assets/translations/en-US.json

```
@@ -2179,6 +2179,7 @@
       "External ID not found": "External ID not found",
       "External ID verified successfully": "External ID verified successfully",
       "External Id": "External Id",
+      "Reversal External Id": "Reversal External Id",
```

Biome (2.5.6)

[error] The key Reversal External Id was already declared.

(lint/suspicious/noDuplicateObjectKeys)